src/esssans/loki/general.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+"""
+Default parameters, providers and utility functions for the loki workflow.
+"""
+
+
+from ..types import (
+    DetectorPixelShape,
+    Incident,
+    LabFrameTransform,
+    LoadedFileContents,
+    NexusDetectorName,
+    NexusInstrumentPath,
+    NeXusMonitorName,
+    NexusSourceName,
+    RunType,
+    TransformationPath,
+    Transmission,
+)
+
+default_parameters = {
+    NexusInstrumentPath: 'instrument',
+    NexusDetectorName: 'larmor_detector',
+    NeXusMonitorName[Incident]: 'monitor_1',
+    NeXusMonitorName[Transmission]: 'monitor_2',
+    NexusSourceName: 'source',
```

Comment:
Some camel-case inconsistencies. One could consider moving the nexus types into a submodule, so we can have `nexus.MonitorName`, `nexus.DetectorName`, etc., but I don't mind too much for now.

Reply:
I changed everything to `NeXus`.